test/etl_test.py

```diff
@@ -131,3 +131,107 @@ def test_ferc1_solo_etl(datastore_fixture,
         pudl_settings_fixture,
         datapkg_bundle_name='ferc1-solo',
         clobber=True)
+
+
+class TestFerc1Datastore:
+    """Validate the Ferc1 Datastore and integration functions."""
+
+    ds = pudl.extract.ferc1.Ferc1Datastore(sandbox=True)
+
+    def test_ferc_folder(self):
+        """Spot check we get correct folder names per dataset year."""
+        assert self.ds.get_folder(1994) == "FORMSADMIN/FORM1/working"
+        assert self.ds.get_folder(2001) == "UPLOADERS/FORM1/working"
+        assert self.ds.get_folder(2002) == "FORMSADMIN/FORM1/working"
+        assert self.ds.get_folder(2010) == "UPLOADERS/FORM1/working"
+        assert self.ds.get_folder(2015) == "UPLOADERS/FORM1/working"
+
+    def test_get_fields(self):
+        """Check that the get fields table works as expected."""
+        expect_path = pathlib.Path(__file__).parent / "data" / "ferc" \
+            / "form1" / "f1_2018" / "get_fields.json"
+
+        with expect_path.open() as f:
+            expect = yaml.load(f.read(), yaml.Loader)
+
+        data = self.ds.get_file(2018, "F1_PUB.DBC")
+        result = get_fields(data)
+        assert result == expect
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=5437498)